app/javascript/crayons/MultiSelectAutocomplete/__stories__/MultiSelectAutocomplete.stories.jsx

```diff
@@ -0,0 +1,42 @@
+import { h } from 'preact';
+import { MultiSelectAutocomplete } from '../MultiSelectAutocomplete';
+
+export default {
+  title: 'BETA/MultiSelectAutocomplete',
+};
+
+export const Default = () => {
+  const options = [
+    'one',
+    'two',
+    'three',
+    'four',
+    'five',
+    'six',
+    'seven',
+    'eight',
+    'nine',
+    'ten',
+  ];
+
+  const fetchSuggestions = (searchTerm) => {
+    const filteredSuggestions = options.filter((option) =>
+      option.startsWith(searchTerm),
+    );
+
+    return Promise.resolve(
+      filteredSuggestions.length === 0 ? [searchTerm] : filteredSuggestions,
+    );
+  };
```

Comment:
Refactor (non-blocking): If the function is async, the value returned is implicitly wrapped as a Promise.

```suggestion
  const fetchSuggestions = async (searchTerm) => {
    const filteredSuggestions = options.filter((option) =>
      option.startsWith(searchTerm),
    );

    return filteredSuggestions.length === 0
      ? [searchTerm]
      : filteredSuggestions;
  };
```